src/helpers/topicPropsBuilder.js

```diff
@@ -115,23 +117,28 @@ function topicPropsBuilder(regEntry, sourceEntry, config) {
 
   // building src and dest locations for release notes
   const releaseNotes = config.docu.folders.releaseNotes;
-  const rnBaseLocation = (type === 'gettingstarted' || type === 'architecture' || type === 'solutions') ? false : `${basicSrcLocation}/${releaseNotes}`;
+  const rnBaseLocation = isWithoutReleaseNotes ? false : `${basicSrcLocation}/${releaseNotes}`;
   const rnSrcLocation = (rnBaseLocation) ? `${rnBaseLocation}/**` : false;
-  const rnBaseLocationInternal = (type === 'gettingstarted' || type === 'architecture' || type === 'solutions') ? false : `${basicSrcLocation}/internal/${releaseNotes}`;
+  const rnBaseLocationInternal = isWithoutReleaseNotes ? false : `${basicSrcLocation}/internal/${releaseNotes}`;
   const rnSrcLocationInternal = (rnBaseLocationInternal) ? `${rnBaseLocationInternal}/**` : false;
-  const rnDestLocation = (type === 'gettingstarted' || type === 'architecture' || type === 'solutions') ? false : `${config.skeletonSrcDestination}/rn/${type}/${shortName}/latest`;
-  const rnDestLocationWithoutVersion = (type === 'gettingstarted' || type === 'architecture' || type === 'solutions') ? false : `${config.skeletonSrcDestination}/rn/${type}/${shortName}`;
-  const rnDestLocationInternal = (type === 'gettingstarted' || type === 'architecture' || type === 'solutions') ? false : `${config.skeletonSrcDestination}/internal/rn/${type}/${shortNameInternal}/latest`;
-  const rnDestLocationInternalWithoutVersion = (type === 'gettingstarted' || type === 'architecture' || type === 'solutions') ? false : `${config.skeletonSrcDestination}/internal/rn/${type}/${shortNameInternal}`;
+  const rnDestLocation = isWithoutReleaseNotes ? false : `${config.skeletonSrcDestination}/rn/${type}/${shortName}/latest`;
+  const rnDestLocationWithoutVersion = isWithoutReleaseNotes ? false : `${config.skeletonSrcDestination}/rn/${type}/${shortName}`;
+  const rnDestLocationInternal = isWithoutReleaseNotes ? false : `${config.skeletonSrcDestination}/internal/rn/${type}/${shortNameInternal}/latest`;
+  const rnDestLocationInternalWithoutVersion = isWithoutReleaseNotes ? false : `${config.skeletonSrcDestination}/internal/rn/${type}/${shortNameInternal}`;
 
   /*
   adding base uri - service proxy url
   */
-  //building env link basing on specified node environment
-  const envLink = envLinkBuilder(config);
+
+  if(!sourceEntry.baseUri && !config.defaultBaseUriDomain){
```

Comment:
delete this error handling since baseUri is always present :) We build it if its not provided by the user :)